content/07.pathogenesis.md

```diff
@@ -46,22 +46,30 @@ One severe example includes reports of cytokine storm-like responses in patients
 
 #### Reproduction Number and Dynamics of Transmission
 
-Accurate estimates of the reproduction number of a virus are crucial to understanding the dynamics of infection and to predict the effects of different interventions.
-The basic reproduction number and the timescale of infection (measured by the infectious period and the exposed period) govern population-level epidemic dynamics, with R~0~ being one of most critical epidemiological parameters [@isbn:9780691116174].
```

Comment:
Note that you can ignore at this point for interest of time.
At this point it just makes sense to go with whatever you have time to go through, but just an optional note I'll re-explain why we had the sentence:
"The basic reproduction number and the timescale of infection (measured by the infectious period and the exposed period) describe population-level epidemic dynamics, with R0 being one of most critical epidemiological parameters [@ISBN:9780691116174]."
It's almost exactly what a very good modern infectious disease modelling textbook says at the very beginning. Every overview talks about R but I was thinking mentioning the timescale of infection would also make sense. If you just have R, all you know about the dynamics is whether the epidemic grows or doesn't (based on if R is greater than 1 or less than 1) without knowing fully how long it takes to grow if it does grow. If you have both R (unitless) & the timescale of infection then you have the dynamics with a scaled notion of time -- i.e. you can describe the dynamics with units. Maybe people just know this as soon as they think through it or read details and they talk about things related to the timescale of infection a lot in other contexts, so that's why it's not important to mention. (It depends on whether you are interested in shape of dynamics only which is major point or also with respect to time units)

Reply:
No I see the importance, we can put it back in :+1: